prov/rxd/src/rxd_cq.c

```diff
@@ -206,51 +209,59 @@ void rxd_ep_recv_data(struct rxd_ep *ep, struct rxd_x_entry *x_entry,
 static void rxd_verify_active(struct rxd_ep *ep, fi_addr_t addr, fi_addr_t peer_addr)
```

Comment:
I know you didn't write this code haha but could you rename addr to be rxd_addr for clarity's sake?

Reply:
this is changed to pass the peer entry instead since the function calling this was also using the same peer_entry.

Reply:
rxd_update_peer(struct rxd_ep *ep, fi_addr_t peer, fi_addr_t peer_addr)  calls this function. so making this change in rxd_update_peer to use rxd_addr as argument name.